pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	svcutil "github.com/aws-controllers-k8s/memorydb-controller/pkg/util"
+)
+
+func (rm *resourceManager) customDescribeACLsSetOutput(
+	ctx context.Context,
+	r *resource,
+	resp *svcsdk.DescribeACLsOutput,
+	ko *svcapitypes.ACL,
+) (*svcapitypes.ACL, error) {
```

Comment:
Just get rid of the `r` and `ko` in the same function arguemnts. You can just using `r` to pick up the name and return the events.

Reply:
This is from required parameter format by `code-generator`, but I can do the way you want as custom code in `templates`.

Reply:
Moved to `sdk_read_many_post_set_output.go.tpl` and modified.